src/web/components/SubNav/SubNav.tsx

```diff
@@ -220,6 +220,7 @@ export const SubNav = ({ subNavSections, format, currentNavLink }: Props) => {
 				<button
 					onClick={() => setIsExpanded(!isExpanded)}
 					className={showMoreStyle}
+					data-link-name="nav2 : subnav-toggle"
```

Comment:
Could be wrong here, but it might be worth sticking a `data-component` on here as well. It should send a component event up to Ophan as well then. I think it depends if it's already declared somewhere else higher up the tree.

Reply:
Thanks. Yeah doesn't look like one on Frontend, but I've added it now as does seem useful.

There's also a `data-cy` attribute, but I'm not sure what that does.

Reply:
I think `data-cy` is used in [Cypress testing](https://docs.cypress.io/guides/references/best-practices.html#How-It-Works) so it would come from that I think